include/olg_scheduler/olg_scheduler.hpp

```diff
@@ -97,6 +97,11 @@ class Event : private cavl::Node<Event<TimePoint>>
     /// It is guaranteed that while an event resides in the tree, it has a valid deadline set.
     [[nodiscard]] std::optional<TimePoint> getDeadline() const noexcept { return deadline_; }
 
+    static constexpr std::array<std::uint8_t, 16> _get_type_id_() noexcept
```

Comment:
This thing needs a comment explaining that it is added for the benefit of cetl::unbounded_variant

Reply:
Added.